SpatialGDK/Source/SpatialGDK/Private/Utils/CommandLineArgs.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Utils/CommandLineArgs.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialCommandLineArgs);
+
+void SpatialGDK::CheckCmdLineOverrideBool(const TCHAR* CommandLine, const TCHAR* Parameter, const TCHAR* PrettyName, bool& bOutValue)
```

Comment:
This has just been moved from SpatialGDKSettings, the only change is the log channel